Podfile

```diff
@@ -1,4 +1,4 @@
-target :RespokeSDKTests, :exclusive => true do
+target :RespokeSDKTests do
```

Comment:
`:exclusive` is no longer supported in cocoapods 1.0, and I don't even really know why it was there to begin with.